### PR TITLE
Nomis Batchloader

### PR DESCRIPTION
**Pull request by atodd-noms**

### CODE REVIEW

File: nomis-batchload/stage/db.tf [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  tags = "${merge(map("Name", "${var.app-name}-db"), var.tags)}"
+}
+
+resource "aws_security_group" "db" {
+  name        = "${var.app-name}-db-sg"
+  vpc_id      = "${aws_vpc.vpc.id}"
+  description = "${var.app-name} DB security group"
+
+  ingress {
+    from_port       = 5432
```
Comment (mattops) [inline nomis-batchload/stage/db.tf]: Do you really want from_port to be 5432?

Comment (Mjwillis) [reply nomis-batchload/stage/db.tf]: yes, that's the postgres port

Comment (mattops) [reply nomis-batchload/stage/db.tf]: it's also the `from_port` ?

Comment (Mjwillis) [reply nomis-batchload/stage/db.tf]: correct from_port and to_port should be the same

Comment (mattops) [reply nomis-batchload/stage/db.tf]: Hang on, read that as source/destination port.  But it's actually a port range for the inbound rule!  Apologies.

**Pull request merged**